apps/frontend/src/libs/components/header/components/user-header-info/user-header-info.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
```suggestion
```

Reply:
After removing the React import, I'm encountering an issue with prop passing, which is leading to errors.
`ESLint: 'user' is missing in props validation(react/ prop-types)`

 Although the following workaround fixes it, it's likely not the correct approach.

`const UserHeaderInformation: React.FC<Properties> = ({ user }: Properties) => {`

And also half of your components have import React.